src/recipes.jl

```diff
@@ -196,6 +200,10 @@ function Makie.plot!(gp::GraphPlot)
         tpx = $to_px(p1) - $to_px(p0)
         atan(tpx[2], tpx[1])
     end
+    # function which projects the pixels to a point in data space
+    to_point = @lift (px) -> begin
+        px ./ ($to_px(Point(1,1)) - $to_px(Point(0,0)))
```

Comment:
I think the `to_point` name and comment is misleading: if I am not mistaken this only contains the "scaling" but not the offset part of the linear transformation between px and dataspace. What about rotation? But might be fine to just change the name, i think it is not needed to have an actual function to go from px to dataspace.

Reply:
Thanks for bringing this up. I don't know how transformations work, so I'm not sure how to address rotation. I will change the name for now if that works...

Reply:
Changed to `scale_px`